lib/game.rb

```diff
@@ -1,21 +1,44 @@
 require_relative 'board'
 require_relative 'console'
+require_relative 'rules'
 
 class Game
-  def initialize(board, console)
+  def initialize(board, console, rules)
     @board = board
     @console = console
+    @current_player = 'X'
+    @rules = rules
   end
 
   def welcome
     @console.print_message('Welcome to Tic Tac Toe')
   end
 
-  def play
-    @console.print_message(@board.display)
-    @console.print_message('Please enter a number 1-9')
+  def turn
+    alert_current_player
+    @console.print_message("Player #{@current_player}, please enter a number 1-9")
     index = @console.retrieve_user_input.to_i - 1
-    @board.position_taken?(index) ? play : @board.mark(index)
+    @board.position_taken?(index) ? turn : @board.mark(index, @current_player)
```

Comment:
What happens if a player selects a number outside the range 1-9?

Reply:
It reruns the turn method and asks the user again to make a move. Would you suggest that the logic of telling the user why they need to try again should be on this ticket?